### PR TITLE
JSON file typings

### PR DESCRIPTION
**Pull request by n-d-r-d-g**
Hi @MrSunshyne,

@k3ii assisted me in generating a `.ts` file for featured-issues.json so we can extract the literal types from it.

What do you think about converting all the other .json files in the same directory to .ts as well?

P.S. There are some .js files as well in the directory. I think we need to convert them to .json & .ts as well.

### CODE REVIEW

File: .github/workflows/featured_issues.yml [hunk 1]
```diff
@@ -22,13 +22,17 @@ jobs:
 
           # Save the issues to a featured-issues.json
           echo "$issues" | jq 'map({number, title, url, author, assignees, createdAt, state, comments_count: .comments | length})' > packages/frontendmu-data/data/featured-issues.json
+          echo "export default " > packages/frontendmu-data/data/featured-issues.ts
+          cat packages/frontendmu-data/data/featured-issues.json | jq '.' >> packages/frontendmu-data/data/featured-issues.ts
```
Comment (MrSunshyne) [inline .github/workflows/featured_issues.yml:26]: @k3ii @n-d-r-d-g 
This looks nice, but probably not maintainable at all. Could we add comments to explain what's going on here? I'm thinking of the scenario when we come back to this 8 months later and don't feel confident to touch it

**Comment by n-d-r-d-g** [comment]
@MrSunshyne alright, @k3ii thought of splitting .json & .ts files in separate directories and creating a script to convert .json to .ts.

@k3ii All .json files in the `data` directory contain arrays, except for `photos-raw.json`. This one needs to be converted slightly differently. We can look into this together.

File: .github/workflows/featured_issues.yml (hunk 1 above)
Comment (n-d-r-d-g) [reply .github/workflows/featured_issues.yml:26]: Definitely not maintainable!
@k3ii will create a separate bash script to perform the conversion. @MrSunshyne You can leave this PR for now I guess.

Comment (k3ii) [reply .github/workflows/featured_issues.yml:26]: Do you want me add comments? @MrSunshyne

**Comment by k3ii** [comment]
> @MrSunshyne alright, @k3ii thought of splitting .json & .ts files in separate directories and creating a script to convert .json to .ts.
> 
> @k3ii All .json files in the `data` directory contain arrays, except for `photos-raw.json`. This one needs to be converted slightly differently. We can look into this together.

@n-d-r-d-g  do we do the work in this PR or  we get this one merge and work on another PR?

**Comment by n-d-r-d-g** [comment]
> > @MrSunshyne alright, @k3ii thought of splitting .json & .ts files in separate directories and creating a script to convert .json to .ts.
> > @k3ii All .json files in the `data` directory contain arrays, except for `photos-raw.json`. This one needs to be converted slightly differently. We can look into this together.
> 
> @n-d-r-d-g do we do the work in this PR or we get this one merge and work on another PR?

@k3ii Yeah maybe it's a better idea to merge this one then. Let's work on another PR for the other changes.

@MrSunshyne could you please merge this one? We'll get rid of the unmaintainable code in the next PR.

**Review by MrSunshyne — APPROVED** [review]

**Pull request merged**